seqr/views/apis/variant_search_api_tests.py

```diff
@@ -209,6 +210,28 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts):
             }
         })
 
+    # Test new search with allProjectFamilies
+    @mock.patch('seqr.views.apis.variant_search_api.get_es_variants')
+    def test_query_variants_all_project(self, mock_get_variants):
+        url = reverse(query_variants_handler, args=[SEARCH_HASH])
+        _check_login(self, url)
+
+        mock_get_variants.side_effect = _get_es_variants
+
+        response = self.client.post(url, content_type='application/json', data=json.dumps({
+            'allProjectFamilies': True, 'search': SEARCH
```

Comment:
I don't see anything in this test that actually tests that the `allProjectFamilies` part of this request is working as intended. If the function completely ignored that parameter, this test would still behave identically. What does that parameter do that is different than if it was absent? Make sure whatever it is, there is something in the test that checks it is doing that. While getting coverage up is important, having tests that cover every line of code but don't actually make any assertions about what those lines do isn't really helpful